### ISSUE TITLE
Get clip/focal settings from active camera

### ISSUE THREAD

**Issue by omgitsraven** [regcs/AliceLG#38]
I have a scene with several cameras, that I want to make separate renders from, and I've just finally noticed that while changing the clip settings in the Looking Glass panel changes them on the camera too, switching to a new camera does NOT update the Looking Glass panel's clip values to match! (Similarly, it doesn't get the focal point value either, but as I noted in a separate issue https://github.com/regcs/AliceLG/issues/29 , that doesn't go in either direction...  but, in light of this, the data should probably go in the *opposite* direction of what I'd requested in that thread!)

Basically, all three values are already stored on the camera that Alice is being attached to, so why does Alice store its own copy rather than just looking at the value set on the current camera? That way I wouldn't have to copy the values over manually every time I change cameras...

**Comment by regcs** [comment]
I understand, that in the use case with multiple cameras the overwriting of camera settings is an inconvenience. I have two problems with changing that though:

1. The standard camera setting of Blender has a clipping from 0.1 to 100, which results in an extremely large view cone, which is not of much relevance for the Looking Glass. 
2. The focal length of the camera has no relevance in normal scenes unless you activate the Depth of Field.

So, the question would be how to handle that? Maybe with an additional option, where the user can choose if the camera settings are overwritten or not?

**Comment by omgitsraven** [comment]
I think an option would work fine, yeah. Or maybe a warning if it's attached to a camera that has Blender's default values, with an option to overwrite them with Alice's defaults instead? (and it could remember whether it's already shown that prompt for any camera in the scene.) Maybe that's overkill though, the option is probably enough.

**Comment by regcs** [comment]
Ok, so then I would opt for a checkbox or something that allows you to toggle on/off the "overwrite camera settings" behavior.

**Comment by regcs** [comment]
Implemented in Alice/LG 2.2.
